### PR TITLE
Auth Middleware

### PR DESCRIPTION
**Pull request by deetz99**
see comments

### CODE REVIEW

File: app/components/Bcros/ContactInfo.vue
Comment (deetz99) [inline app/components/Bcros/ContactInfo.vue:1]: fix styling that is overwritten from the new core layer version

File: app/components/PPR/ContactInfo.vue
Comment (deetz99) [inline app/components/PPR/ContactInfo.vue:1]: fix styling that is overwritten from the new core layer version

File: app/components/Sbc/AuthMenu.vue
Comment (deetz99) [inline app/components/Sbc/AuthMenu.vue:1]: fix styling that is overwritten from the new core layer version

File: app/components/Sbc/I18nBold.vue
Comment (deetz99) [inline app/components/Sbc/I18nBold.vue:1]: moved this component to the core layer

File: app/components/Sbc/LoadingSpinner.vue
Comment (deetz99) [inline app/components/Sbc/LoadingSpinner.vue:1]: moved this component to the core layer

File: app/components/Sbc/PageSection/Card.vue
Comment (deetz99) [inline app/components/Sbc/PageSection/Card.vue:1]: moved this component to the core layer

File: app/components/Sbc/Tooltip.vue
Comment (deetz99) [inline app/components/Sbc/Tooltip.vue:1]: moved this component to the core layer

File: app/layouts/bcreghome.vue
Comment (deetz99) [inline app/layouts/bcreghome.vue:1]: tossing this and just overwriting the default layout

File: app/pages/[...404].vue
Comment (deetz99) [inline app/pages/[...404].vue:1]: catch-all route, maybe switch to error.vue ???

File: app/pages/auth/signin.vue
Comment (deetz99) [inline app/pages/auth/signin.vue:1]: generic sign in page to redirect auth routes to in middleware

File: app/pages/dashboard.vue
Comment (deetz99) [inline app/pages/dashboard.vue:1]: middleware and breadcrumbs, plus clientonly but still laggy

File: app/plugins/auth-api.ts
Comment (deetz99) [inline app/plugins/auth-api.ts:1]: moved this to the core layer

File: tests/unit/components/SbcI18nBold.test.ts
Comment (deetz99) [inline tests/unit/components/SbcI18nBold.test.ts:1]: moved this component to the core layer

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**